src/Aquifer.API/Endpoints/Projects/ProjectResourceStatusCountHelper.cs

```diff
@@ -0,0 +1,64 @@
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.API.Endpoints.Projects;
+
+public static class ProjectResourceStatusCountHelper
+{
+    private static async Task<List<ProjectResourceSourceWordCount>> GetCountsPerProjectAsync(
+        IEnumerable<int> projectIds,
+        AquiferDbContext dbContext,
+        CancellationToken ct)
+    {
+        var query = $"""
+                     SELECT
+                         SUM(RCVD.SourceWordCount) AS WordCount,
+                         RC.Status,
+                         PRC.ProjectId
+                     FROM ResourceContents RC
+                     JOIN ProjectResourceContents PRC ON RC.Id = PRC.ResourceContentId
+                     LEFT JOIN
+                     (
+                         SELECT *
+                         FROM
+                         (
+                             SELECT
+                                 *,
+                                 ROW_NUMBER() OVER (PARTITION BY ResourceContentId ORDER BY [Version] DESC) AS LatestVersionRank
+                             FROM ResourceContentVersions
+                         ) x
+                         WHERE x.LatestVersionRank = 1
+                     ) RCVD ON RCVD.ResourceContentId = RC.Id
+                     WHERE PRC.ProjectId IN ({string.Join(", ", projectIds)})
```

Comment:
I experienced a lot of query performance pain this week because of SQL query plans being different when different SQL query strings are passed to SQL server.  By injecting the `projectIds` directly into the SQL query text, SQL Server thinks that the queries are entirely different if any project IDs are different because the SQL query strings don't match.  If SQL parameters are used (e.g. `WHERE PRC.ProjectId IN @projectIds`) then it will use the same query plan because the SQL query string remains the same.

I'm not suggesting changing anything right now for this PR (there are existing endpoints that do this same thing) but I'm happy to demo how to use Dapper some time which allows passing `IEnumerable<T>` parameter arguments, unlike `SqlQueryRaw().